packages/devtools/lib/src/timeline/cpu_profile_model.dart

```diff
@@ -25,6 +25,7 @@ class CpuProfileData {
       category: 'Dart',
       url: '',
       profileTime: time,
+      profileSampleCount: sampleCount,
```

Comment:
for this variable and the profileTime one above remove the word "profile" from the name to keep the names clean. The class name already indicates that this is a profile related object..